pom.xml

```diff
@@ -31,7 +31,7 @@
     </scm>
 
     <properties>
-        <graalvmHome>${env.GRAALVM_HOME}</graalvmHome>
+        <graalvmHome>$GRAALVM_HOME</graalvmHome>
```

Comment:
Why were these changed?

Reply:
It was giving some error, I changed these before the first commit i guess, should I undo this?

Reply:
Yes please

Reply:
Okk